Sources/UtilityBeltNetworking/Models/MimeType.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+public struct MimeType: RawRepresentable, Codable {
```

Comment:
@bdrelling this is the only way I found to make a struct act like a string enum, such that a user of `UtilityBeltNetworking` could add extra "cases" in an extension.  Doesn't seem right tho.... Is there a different protocol to use here that would accomplish this better?

Reply:
No, this looks totally great! What do you think doesn't seem right about it? I assumed the protocol was `RawStringConvertible` or something but this actually seems more appropriate.

The only way you could make it simpler is by incorporating `ExpressibleByStringLiteral`, but I don't mind how you have it right now honestly.

Reply:
Ah I thought you had a slicker way to do it with just string literals, not using raw values.